addons/sales_team/tests/test_default_team.py

```diff
@@ -0,0 +1,40 @@
+from odoo.tests import common
+
+
+class TestDefaultTeam(common.SavepointCase):
+    """Tests to check if correct default team is found."""
+
+    @classmethod
+    def setUpClass(cls):
+        """Set up data for default team tests."""
+        super(TestDefaultTeam, cls).setUpClass()
+        cls.CrmTeam = cls.env['crm.team']
+        # America Team
+        cls.team_1 = cls.env.ref('sales_team.crm_team_1')
+        # Europe Team (last resort default team)
+        cls.team_2 = cls.env.ref('sales_team.team_sales_department')
+        cls.user_demo = cls.env.ref('base.user_demo')
+        # Make user belong to America team, so it would be different
+        # from last resort default team.
+        cls.team_1.member_ids = [(4, cls.user_demo.id)]
+
+    def test_default_team_1(self):
```

Comment:
Use more meaningful test names please, and prefix them with a sequence
E.g. `test_01_user_team` and `test_02_fallback_team`